src/main/java/domain/state/StateFactory.java

```diff
@@ -0,0 +1,25 @@
+package domain.state;
+
+import domain.card.Card;
+import domain.card.Hands;
+
+public class StateFactory {
```

Comment:
정적 메서드만 포함하고 있는 유틸리티 클래스라면 인스턴스 생성을 방지하기 위해 아래와 같은 코드를 추가해보는건 어떨까요?
```suggestion
public class StateFactory {

    private StateFactory() {
    }
```